File: tools/commonist/src/scala/commonist/ui/NotWideComboBox.scala
package commonist.ui

import java.awt.Dimension

import javax.swing.JComboBox

import commonist.data.LicenseData

// TODO make this a trait
class NotWideComboBox(data:Array[Object]) extends JComboBox(data) {
	override def getPreferredSize():Dimension = new Dimension(
			10,
			super.getPreferredSize.height)
}

File: tools/commonist-java/src/net/psammead/commonist/ui/NotWideComboBox.java
package net.psammead.commonist.ui;

import java.awt.Dimension;
import java.util.Vector;

import javax.swing.JComboBox;

import net.psammead.commonist.data.LicenseData;

class NotWideComboBox extends JComboBox {
	public NotWideComboBox(Vector<LicenseData> vector) {
		super(vector);
	}

	@Override
	public Dimension getPreferredSize() {
		return new Dimension(
				10,
				super.getPreferredSize().height);
	}
}